packages/zkdb/src/sdk/impl/collection.ts

```diff
@@ -172,13 +172,17 @@ export class CollectionQueryImpl implements ZKCollection {
     },
   >(model: InstanceType<T>, permissions?: Permissions): Promise<MerkleWitness> {
```

Comment:
consider in case permission is {} (empty object) the code will break

Reply:
Works fine

Reply:
really? If the empty object existed on documentPermission, is it normal?
```ts
 documentPermission: {
        permissionOwner: undefined,
        permissionOther: undefined,
        permissionGroup: undefined,
      },
```

Reply:
I need to investigate why the permissions are no longer being sent to the server

Reply:
It sometimes happens when frontend dev is missing to pass or in runtime, it can be {}. It would help if you did not trust that code always gives you the best case.

Reply:
done